### ISSUE TITLE
login in localhost:8333

### ISSUE THREAD

**Issue by sanibbi** [LLK/scratch-www#3095]
what happens when I try to signin the localhost? It seems that nothing happens when I put the username/password in the loin page in the localhost.

**Comment by apple502j** [comment]
yes, it should not work. Modified version of Scratch should not affect the original one.
